src/components/pages/rules/flyouts/ruleEditor/ruleEditor.js

```diff
@@ -304,7 +371,24 @@ export class RuleEditor extends LinkedComponent {
       return { fieldLink, operatorLink, valueLink, error };
     });
 
+    this.newEmailLink = this.linkTo('newEmail') // Matches email address pattern
```

Comment:
Since this link is not dynamic, it is better to set it in the constructor so it doesn't need to be recreated every time render is fired.